nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/osgi/OSGiComponentLoader.java

```diff
@@ -119,6 +119,7 @@ public void bundleChanged(BundleEvent event) {
                 break;
             }
         } catch (RuntimeException e) {
+            // TODO review: should we throw the exception ? this will allow to break dev hotreload properly (and tests)
             log.error(e, e);
```

Comment:
I'd say each bundle should be independent from the others, so should not block others.